metadata-ingestion/src/datahub/ingestion/source/sql/sql_generic_profiler.py

```diff
@@ -267,14 +273,14 @@ def is_dataset_eligible_for_profiling(
                 self.config.profiling.profile_if_updated_since_days
             )
 
-        if not self.config.profile_pattern.allowed(dataset_name):
-            return False
-
         schema_name = dataset_name.rsplit(".", 1)[0]
         if (threshold_time is not None) and (
             last_altered is not None and last_altered < threshold_time
         ):
             self.report.profiling_skipped_not_updated[schema_name] += 1
+            logger.debug(
+                f"Table {dataset_name} is not allowed for profiling due to it was not updated recently enough"
```

Comment:
"is not allowed" -> "was skipped"

should make these logs more clear

```suggestion
                f"Table {dataset_name} was skipped because it was not updated recently enough"
```